### DISCUSSION TITLE
how to start newly libre 2 sensor via code ??

### DISCUSSION THREAD

**Discussion by hiral101** [NightscoutFoundation/xDrip#2050] [General]
how to start newly sensor ?? can you please help me ?? because I think their no any code for start the new sensor in xdrip app

**Comment by Navid200** [comment]
@psonnera

**Reply by psonnera** [reply]
I've never seen a code request for Libre Bluetooth data source at sensor start. Physically Libre 2 sensors can only be started by the vendor app or the vendor reader. @hiral101 can you post a screenshot? Which data source are you using?

**Comment by tzachi-dar** [comment]
You need to start it with a libre reader or with librelink app.
